setup.cfg

```diff
@@ -47,7 +47,7 @@ install_requires =
     toml>=0.10.2
     tqdm>=3.4.0
     unyt>=2.8.0
-python_requires = >=3.6
+python_requires = >=3.6,<3.12
```

Comment:
To anyone wondering: the reasoning here is that the distutils builtin package is slanted for removal in Python 3.12 (which should be released around October 2023), but it is currently used in several places in yt, making it de facto incompatible with Python 3.12
Solving this doesn't seem like a priority but I'll document it as an issue !